onnxruntime/contrib_ops/rocm/bert/skip_layer_norm_impl.cu

```diff
@@ -41,9 +42,18 @@ namespace rocm {
 template <typename T>
 Status LaunchSkipLayerNormKernel(
```

Comment:
Can we also refactor this function, `LaunchFooBarKernel` should only wrap kernel launch logic for `__global__ void FooBar(...)`. The logic for switching between tuning and non-tuning should sit outside of this function.